s14/lambda_example.rb

```diff
@@ -0,0 +1,36 @@
+def convert_to_euros(dollars)
+  dollars * 0.95 if dollars.is_a?(Numeric)
+end
+def convert_to_pesos(dollars)
+  dollars * 20.67 if dollars.is_a?(Numeric)
+end
+def convert_to_rupees(dollars)
+  dollars * 68.13 if dollars.is_a?(Numeric)
+end
+p convert_to_euros(1000)
+p convert_to_pesos(1000)
+p convert_to_rupees(1000)
+puts
+
+
+def convert(dollars, currency)
+  yield(dollars, currency) if dollars.is_a?(Numeric)
```

Comment:
Using a Block, an argument is sent to a method that hasn't input parameters, so that’s where the word yield is used, because where it is located is where the argument will enter.